tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"declaration": true,
+		"declaration": false,
```

Comment:
This is due to the `chalk` update meaning that lazy-loading fails because it can't re-export private types from chalk for declaration files. Since the cli is not intended to be used as a module the declaration types aren't useful and disabling them avoids the issue

Reply:
Just realised... This would mean that the `d.ts` files would no longer be created in the `build` folder, right? A consequence might be that the CLI test files written in Typescript would no longer be able to perform type checking, because they import directly from the build folder. For example:
https://github.com/balena-io/balena-cli/blob/v11.28.13/tests/helpers.ts#L26

What could we do to preserve type information in tests, one way or another?

Reply:
Just realised... This would mean that the `d.ts` files would no longer be created in the `build` folder, right? A consequence might be that the CLI test files written in Typescript would no longer be able to perform type checking, because they import directly from the build folder. For example:
https://github.com/balena-io/balena-cli/blob/v11.28.13/tests/helpers.ts#L26

What could we do to preserve type information in tests, one way or another?